projects/admin/src/app/service/menu.service.ts

```diff
@@ -90,12 +91,30 @@ export class MenuService {
     ]
   };
 
+  /**
+   * Constructor
+   * @param _translateService : TranslateService
+   * @param _userService : UserService
+   */
   constructor(
     private _translateService: TranslateService,
     private _userService: UserService,
   ) {}
 
+  /** Router link to my library */
   private _myLibraryRouterLink() {
     return `/records/libraries/detail/${this._userService.getCurrentUser().currentLibrary}`;
   }
+
+  /** Router link to my organisation */
+  private _myOrganisationRouterLink() {
+    return `/records/organisations/detail/${this._userService.getCurrentUser().library.organisation.pid}`;
+  }
+
+  /** Query params to filter documents by organisation
```

Comment:
Doc blocks must be written as this (empty line before and after method description)

```typescript
/**
 * Query params to filter documents by organisation
 *  
 * @return organisation pid as a dictionary
 */
```